### PR TITLE
[AUD-1220] Add share-to-twitter action to ShareDrawer

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Adds share-to-twitter action to ShareDrawer

### Dragons

Should we include analytics similar to https://github.com/AudiusProject/audius-client/blob/70748bba8f05cc9f15adf0d38b5d747d6d0f5ffa/src/containers/notification/components/TwitterShare.tsx#L278 ?

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-aud-1220-add-share-to-twitter

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-aud-1220-add-share-to-twitter

File: src/containers/share-modal/components/ShareDrawer.tsx [hunk 1]
```diff
@@ -17,12 +21,27 @@ export const ShareDrawer = () => {
...
   return (
     <ActionDrawer
       title='Share Track'
       actions={[
+        {
+          text: 'Share to Twitter',
+          icon: <IconTwitterBird height={20} width={26} />,
+          className: styles.shareToTwitterAction,
+          onClick: () => {
+            if (track && artist) {
```
Comment (raymondjacobson) [inline src/containers/share-modal/components/ShareDrawer.tsx:38]: might be worth an else that does a console.error. we started piping those to sentry recently, which is nice!

**Review by raymondjacobson — APPROVED** [review]

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-aud-1220-add-share-to-twitter

**Review by rickyrombo — APPROVED** [review]
LGTM! I like the idea of including the share event to the metrics, though, if you want to include that

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-aud-1220-add-share-to-twitter

**Pull request merged**